src/utils/input/utils.js

```diff
@@ -35,28 +35,51 @@ export const splitSource = (source) => {
 	};
 };
 
-export const getStdIn = ({ dataIsRequired, passphraseIsRequired } = {}) =>
+export const getStdIn = ({
+	passphraseIsRequired,
+	secondPassphraseIsRequired,
+	passwordIsRequired,
+	dataIsRequired,
+} = {}) =>
 	new Promise((resolve) => {
-		if (!dataIsRequired && !passphraseIsRequired) return resolve({});
+		if (!(
+			passphraseIsRequired
+			|| secondPassphraseIsRequired
+			|| passwordIsRequired
+			|| dataIsRequired
+		)) {
+			return resolve({});
+		}
 
 		const lines = [];
 		const rl = readline.createInterface({ input: process.stdin });
 
-		const handleLine = line => (
-			dataIsRequired
-				? lines.push(line)
-				: resolve({ passphrase: line }) || rl.close()
-		);
 		const handleClose = () => {
-			const dataLines = passphraseIsRequired ? lines.slice(1) : lines;
+			const passphrase = passphraseIsRequired ? lines[0] : null;
+
+			const secondPassphraseIndex = +(passphrase !== null);
```

Comment:
missing `passphraseIndex` ?

Reply:
It makes no functional difference, but probably improves intelligibility.